terraform/app/codedeploy.tf

```diff
@@ -116,8 +116,8 @@ resource "aws_s3_object" "appspec_object" {
   key    = "appspec.yaml"
   acl    = "private"
   content = templatefile("templates/appspec.yaml.tpl", {
-    task_definition_arn = aws_ecs_task_definition.task_definition.arn
-    container_name      = jsondecode(aws_ecs_task_definition.task_definition.container_definitions)[0].name
+    task_definition_arn = module.web_service.task_definition.arn
+    container_name      = module.web_service.task_definition.arn
```

Comment:
If the variables are the same, can we remove the `container_name` from the template?

Reply:
It's changed again in the following PR https://github.com/nhsuk/manage-vaccinations-in-schools/pull/3323